packages/common/src/ui/layout/tables/context/TableContext.ts

```diff
@@ -172,6 +172,7 @@ export const createTableStore = (): UseStore<TableStore> =>
 
         // Get currently focused row, if any
         const [currentFocusId, currentFocusObj] =
+          // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
what is unused here?

Reply:
it doesn't like the first parameter in `rows.find(([_, { isFocused }])...` not being used. 
have realised belatedly that I can just use `rows.find(([, { isFocused }])...` so have removed the eslint hint. 🙄

Reply:
strange since '_' usually makes the linter happy